### ISSUE TITLE
[Dashabord] Message requeue error

### ISSUE THREAD

**Issue by kingflyok** [dotnetcore/CAP#205]
An exception occurs when requeue the messages at published & received message list on the dashboard.

**exception stack :**
```
fail: Microsoft.AspNetCore.Server.Kestrel[13]
      Connection id "0HLGPRU6PVQ2S", Request id "0HLGPRU6PVQ2S:00000006": An unhandled exception was thrown by the application.
System.OverflowException: Value was either too large or too small for an Int32.
   at System.Number.ParseInt32(String s, NumberStyles style, NumberFormatInfo info)
   at DotNetCore.CAP.Dashboard.BatchCommandDispatcher.<Dispatch>d__2.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Hosting.Internal.RequestServicesContainerMiddleware.<Invoke>d__3.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.Frame`1.<ProcessRequestsAsync>d__2.MoveNext()
```

**Comment by yang-xiaodong** [comment]
This is a bug for version 2.3!
Thank you for your feedback

**Comment by kingflyok** [comment]
谢谢，感谢大神的回复

**Comment by yang-xiaodong** [comment]
fixed in `2.3.1-preview-53660607`
